config/locales/th.yml

```diff
@@ -0,0 +1,341 @@
+---
+en:
```

Comment:
Looks like a typo here, could you change this to be the Thai language shortcode?